Utilities/Logging.cs

```diff
@@ -104,6 +104,41 @@ private static void log(ErrorLevel errorlevel, string data, string method, strin
             }
         }
 
+        public static void incrementLogs()
+        {
+            // Obtain files, sorting by last write time to ensure that older files are incremented prior to newer files
+            DirectoryInfo directoryInfo = new DirectoryInfo(Constants.DATA_DIR);
+            foreach (FileInfo file in directoryInfo.GetFiles().OrderBy(f => f.LastWriteTimeUtc).ToList())
```

Comment:
Consider EnumerateFiles("*.log") for a possible performance improvement. It can't hurt and often the OS and kernel canb make use of the hint to perform optimisations that we don't know about.

Reply:
Elaborate? `EnumerateFiles()` does not appear to be part of the standard libraries.

Reply:
Method on `Directory` and `DirectoryInfo`.
https://docs.microsoft.com/en-us/dotnet/standard/io/how-to-enumerate-directories-and-files